canonical-web-and-design/juju.is
# Add page at /operator-day with videos from past events

## Done

- Added /operator-day page based on [copy doc](https://docs.google.com/document/d/1ZFRGkZ4vvvMVOgq4MIsBuebHsA7Y0rTyiZ76qjQpOAg/edit#heading=h.rbuwfvimc6g8)
- It's very barebones, I spoke to [@tmihoc](https://github.com/tmihoc) about it and they're happy for this to go up for the event next week, then iterate when they have time to add more content

## QA

- visit /operator-day
- See that it matches the [copy doc](https://docs.google.com/document/d/1ZFRGkZ4vvvMVOgq4MIsBuebHsA7Y0rTyiZ76qjQpOAg/edit)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5273
